app/src/main/java/com/internship/move/feature/authentication/register/RegisterFragment.kt

```diff
@@ -15,16 +20,56 @@ class RegisterFragment : Fragment(R.layout.fragment_register) {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        initViews()
         initListeners()
     }
 
     private fun initListeners() {
-        binding.goHomeBtn.setOnClickListener {
+        binding.launchHomeBtn.setOnClickListener {
             findNavController().navigate(RegisterFragmentDirections.actionRegisterFragmentToMapFragment())
         }
 
-        binding.loginBtn.setOnClickListener {
+        binding.goToLoginTV.addClickableText(getString(R.string.launch_login_link))
+        {
             findNavController().navigate(RegisterFragmentDirections.actionRegisterFragmentToLoginFragment())
         }
+
+        binding.termsAndConditionsTV.addClickableText(getString(R.string.t_and_c_link))
```

Comment:
In kotlin, we usually open the brackets on the same line.
so instead of 
```
if (...)
{
..
}
```

we do 
```
if (...) {
...
}
```